debian/rules

```diff
@@ -169,9 +169,35 @@ install: build
 	for i in `find * -path 'docs/images/*' -type f`; do \
 		install -m0644 $$i $(CURDIR)/debian/packetfence-doc$(PREFIX)/$(NAME)/html/pfappserver/root/static/images/; \
 	done
+	#Portal javascript/css
+	make -C html/common/ vendor
+	make -C html/common light-dist
```

Comment:
We also need grunt (`sudo npm install -g grunt-cli`). Should we install it from the Makefile?

Reply:
I did it in the chroot directly but i also changed in the Makefile